homeassistant/components/plugwise/climate.py

```diff
@@ -92,15 +98,31 @@ def __init__(self, api, name, min_temp, max_temp):
         self._name = name
         self._domain_objects = None
         self._outdoor_temperature = None
-        self._active_schema = None
+        self._selected_schema = None
         self._preset_mode = None
-        self._hvac_modes = ATTR_HVAC_MODES
+        self._presets = None
+        self._presets_list = None
+        self._heating_status = None
+        self._cooling_status = None
+        self._schema_names = None
+        self._schema_status = None
+        self._current_temperature = None
+        self._thermostat_temperature = None
+        self._illuminance = None
+        self._boiler_temperature = None
+        self._water_pressure = None
+        self._schedule_temperature = None
+        self._hvac_mode = None
+        self._hvac_modes_1 = ATTR_HVAC_MODES_1
```

Comment:
We don't need to store module level constants as instance attributes. We can use the constants directly.

Reply:
Implemented accordingly.